cpp/python/src/python_future_task_collector.cpp

```diff
@@ -34,7 +34,8 @@ void PythonFutureTaskCollector::collect()
     std::lock_guard<std::mutex> lock(_mutex);
     for (auto& handle : _toCollect)
       Py_XDECREF(handle);
-    ucxx_trace("Collected %lu PythonFutureTasks", _toCollect.size());
+    ucxx_trace("PythonFutureTaskCollector::collect, collected %lu PythonFutureTasks",
```

Comment:
Might be missing some nuance of how this plugs into the Python/Cython API, would we want to fully qualify this?


```suggestion
    ucxx_trace("ucxx::python::PythonFutureTaskCollector::collect, collected %lu PythonFutureTasks",
```

Reply:
Same as above, but fixed it the same way in https://github.com/rapidsai/ucxx/pull/156/commits/f9fa937fc23b34e9aa179b8727fe93881ee763e1 as well.